tests/garage/torch/policies/test_continuous_mlp_policy.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+from unittest.mock import MagicMock
+
+import akro
+import numpy as np
+import torch
+from torch import nn
+
+from garage.torch.modules import MLPModule
+from garage.torch.policies import ContinuousMLPPolicy
+from tests.fixtures.envs.dummy import DummyBoxEnv
+
+
+class TestContinuousPolicies(unittest.TestCase):
+    def test_output_1d(self):
+        env_spec = MagicMock(return_value=DummyBoxEnv())
+        env_spec.action_space = MagicMock(
```

Comment:
Why do you need `MagicMock` here? Is the existing test fixture in `DummyBoxEnv` missing some functionality?

Reply:
honestly i think Mocks are better than the fixtures, unless it is overly-cumbersome to use mock for this case.

in any case, yes, it makes no sense to mock a fixture.